tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -207,14 +207,23 @@ spec:
 			Expect(stdOut).To(ContainSubstring("example"))
 			Expect(stdOut).To(ContainSubstring("EtcdCluster"))
 
+			// now check for json output
+			jsonOut := helper.CmdShouldPass("odo", "service", "list", "-o", "json")
+			Expect(jsonOut).To(ContainSubstring("\"apiVersion\": \"etcd.database.coreos.com/v1beta2\""))
+			Expect(jsonOut).To(ContainSubstring("\"kind\": \"EtcdCluster\""))
```

Comment:
What do you think if you use https://github.com/openshift/odo/blob/master/tests/helper/helper_generic.go#L73 instead of asserting it 3 times.